test/data/template_recordings/builtin_class_synopsis.rst

```diff
@@ -340,6 +342,15 @@ Member Function Description
 
 
 
+-----
+
+.. _project::v1_0_0::coffee::machine::get_bean_count(mug_sizesize_of_mug)const:
+
+| uint32_t **get_bean_count** (mug_size size_of_mug)
```

Comment:
I don't think we should "rewrite" the function like this. It should be: 
``` 
auto get_bean_count(mug_size size_of_mug) const -> uint32_t;
```
Like in the source code. Otherwise people may not be able to find the function if the search the code base for it etc.